### PR TITLE
Adjust location and wording of testing and publishing

### LINKED ISSUE #3799: Extensions Documentation: Adjust Location and Wording of Testing and Publishing
As I began developing my first VS Code extension, I worked sequentially through the [Extensions documentation](https://marketplace.visualstudio.com/VSCode). 

[Working with Extensions](https://code.visualstudio.com/api/working-with-extensions/testing-extension#working-with-extensions-articles) is one of the first topics, which I had thought was because it would describe how to add greater functionality to our "Hello World" extension. I didn't anticipate it would instead explain testing and publishing.
 
**Potential Changes to Working with Extensions**
- Move "Working with Extensions" to just before or after [Advanced Topics](https://code.visualstudio.com/api/references/commands#advanced-topics-articles).
     - Once you've created an extension and added polish, you're more ready to publish and test.
     - The "Publishing Extensions" article starts with, "Once you have made a high-quality extension, you can publish it," so it would make sense to move this topic to a point where users are more likely to have a polished extension.
- Rename the topic to "Testing and Publishing Extensions" 
     - The current title "Working with Extensions" sounds a bit more like adding components to an extension or implementing extensions in various apps.
- Potentially reorder the articles within the topic.
     - Proposed order: "Publishing Extensions," "Bundling Extensions," "Testing Extensions," "Continuous Integration"
     - Next steps listed in "Publishing" article: "Testing" and "Bundling"
     - Next steps listed in "Bundling" article: "Testing" and "Continuous"
     - Next steps listed in "Testing" article: "Continuous Integration"
     - "Continuous" article doesn't have any next steps, seems to make sense as last.

The changes to "Working with Extensions" could then lead to a slight change in the [Overview](https://code.visualstudio.com/api) article that starts off the extension documentation.
 
**Potential Changes to Overview**
- Change order of first bulleted list to move testing and publishing to the end, such as:

> This documentation describes:
> •	How to build and run an extension
> •	How to take advantage of VS Code's rich Extension API
> •	Where to find guides and code samples to help you get started
> •	How to test and publish your finished extension
 
- Adjust the wording and order of the topics under [How to build extensions?](https://code.visualstudio.com/api#how-to-build-extensions)
     - Rename "Working with Extensions" to "Testing and Publishing Extensions"
     - Move "Testing and Publishing Extensions" toward end of bulleted list
 
Modifying [Working with Extensions](https://code.visualstudio.com/api/working-with-extensions/testing-extension#working-with-extensions-articles) would also impact the [Wrapping Up]( https://code.visualstudio.com/api/get-started/wrapping-up) article in the "Get Started" topic.
 
**Potential Changes to Get Started: Wrapping Up**
- Adjust name of "Working with Extensions" to "Testing and Publishing Extensions," and/or remove this section as a next step all together.
     - Now that I only have a "Hello world" level extension, I wouldn't necessarily think of testing and publishing as the next steps I'd want to take- I want to add more functionality first. 
     - Could remove it, or move it after "Extension Capabilities," since I'd consider it more advanced than "Extension Capabilities."

### PR DESCRIPTION
**Pull request by bamurtaugh**
For additional context, please see #3799 

**Changes to "[Working with Extensions](https://code.visualstudio.com/api/working-with-extensions/testing-extension#working-with-extensions-articles)":**
- Rename "Working with Extensions" to "Testing and Publishing" 
     - Considered "Testing and Publishing Extensions," but the maximum length of any other TOC titles was 3 words, and it seems implied that users would be testing and publishing extensions in these articles.
- Move "Working with Extensions" just before [Advanced Topics](https://code.visualstudio.com/api/references/commands#advanced-topics-articles).
     - Once you've created an extension and added polish, you're more ready to publish and test.
     
**Changes to [Overview](https://code.visualstudio.com/api):**
- Adjust the wording and order of the topics under [How to build extensions?](https://code.visualstudio.com/api#how-to-build-extensions)
     - Rename "Working with Extensions" to "Testing and Publishing"
     - Move "Testing and Publishing" toward end of bulleted list
 
**Changes to [Wrapping Up]( https://code.visualstudio.com/api/get-started/wrapping-up) article in the "Get Started" topic:**
- Adjust name of "Working with Extensions" to "Testing and Publishing"
- Move to the end of article

Fixes #3799

### CODE REVIEW

**Comment by gregvanl** [comment]
@bamurtaugh I'd prefer we didn't change the location of the Working with Extensions topics since that will change the URL and we'll need to add redirects for each topic. I like the changes to the TOC name and ordering and we can do those would affecting the topic URLs.

**Comment by bamurtaugh** [comment]
@gregvanl Thanks for the feedback! If I'm understanding you correctly, it would be okay to adjust the TOC name users see and order the topic differently so that it appears above Advanced, but you would prefer we didn't change the URLs (so behind the scenes, they would remain part of "working-with-extensions" to avoid changing URLs and redirecting users, eliminating changes like the ones below)? 

image

**Comment by gregvanl** [comment]
@bamurtaugh Exactly. You can make the TOC updates in toc.json (rename, reorder) and revert the file moves and any path updates to relative file and image paths.

**Comment by bamurtaugh** [comment]
@gregvanl Got it, thanks for clarifying. I made those updates now so that only the TOC has been updated, along with the wording in Overview and Wrapping Up.

File: api/get-started/wrapping-up.md [hunk 1]
```diff
@@ -11,18 +11,18 @@ MetaDescription: Next steps to take after studying the Getting Started section
...
 
 In this section, we split the [VS Code API](/api/references/vscode-api) and [Contribution Points](/api/references/contribution-points) into a few categories, each with short descriptions as to what your extension could achieve. Validate that your extension idea is achievable by reviewing the [VS Code API](/api/reference/vscode-api) or reading the [Extension Capabilities](/api/extension-capabilities/overview) section for new extension ideas.
 
 ## Guides & Samples
 
 We have a great collection of sample extensions that you can adapt from, and some of them include a detailed guide that explains the source code. You can find all Samples & Guides in the [Extension Guide Listing](/api/extension-guides/overview) or the [vscode-extension-samples](https://github.com/Microsoft/vscode-extension-samples) repository.
+
+## Testing and Publishing
+
+This section includes topics that help you develop high-quality VS Code extension. For example, you can learn
```
Comment (ornellaalt) [inline api/get-started/wrapping-up.md:24]: Typo: extension**s** should be plural

Comment (bamurtaugh) [reply api/get-started/wrapping-up.md]: Thanks for pointing this out, fixed.

**Comment by ornellaalt** [comment]
Thank you!

**Pull request merged**